man/assign.Rd

```diff
@@ -18,6 +18,9 @@
 #            LHS2 = RHS2,
 #            ...), by = ...]
 
+# 3. Multiple columns in place
+# DT[i, .SD = lapply(.SD, fx), by = ..., .SDcols = ...]
```

Comment:
There is typo, you wanted `.SD := lapply`, not `.SD = lapply`. I actually prefer to support only `names(.SD)`, but will comment on that separately.